internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,47 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("ожидается 2, а получено %v", len(parts))
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("не верный формат шагов %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("не верный формат шагов %w", err)
```

Reply:
Нужно везде поправить